app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @Christabelle143 great work on your association, however, to make the relationship complete kindly add the foreign key that links the comment to a particular user using class name eg belongs_to :author, class_name: 'Model name', also change the belong to from user to author, this will make it easier to implement devise in your project.